packages/perseus/src/widgets/label-image.test.ts

```diff
@@ -319,544 +319,6 @@ describe("LabelImage", function () {
         });
     });
 
-    describe("navigateToMarkerIndex", function () {
```

Comment:
Phew, were all these tests verifying things that the WB components handle now?

Reply:
This actually just doesn't work, but I meant to un-delete this since I left the non-working code in there (trying to control unnecessary scope creep where I can)

Reply:
It doesn't work in prod either, not just in the PR code